Composer/packages/extensions/obiformeditor/src/Form/fields/styles.css

```diff
@@ -54,3 +54,7 @@
 a {
   color: black;
 }
+
+input::placeholder {
+  font-style: italic;
+}
```

Comment:
Let's not make this a global style. Also, moving forward, only use css-in-js

Reply:
![image](https://user-images.githubusercontent.com/5860999/69237901-a6a31f00-0bd1-11ea-8139-0f3e630f5e09.png)
It should be a global style according to @sangwoohaan